drf_tweaks/autodoc.py

```diff
@@ -113,12 +113,6 @@ class VersioningAutodoc(AutodocBase):
 
     @classmethod
```

Comment:
please remove all yaml parts from the code (and docs), since they are obsolete anyway